apps/reference/docs/guides/database/migrating-between-projects.mdx

```diff
@@ -0,0 +1,97 @@
+---
+id: migrating-between-projects
+title: 'Migrating Between Projects'
+description: Migrate data from one project to another
+---
+
+Migrating projects can be achieved using standard PostgreSQL tooling. This is particularly useful for older projects (e.g. to use a newer Postgres version).
```

Comment:
```suggestion
Use standard PostgreSQL tooling to migrate data from one Supabase project to another. This is useful for older projects (e.g., to use a newer Postgres version).
```

Reply:
@dannykng I think the initial phrasing was to indicate that no custom tooling is required to do the migration and standard postgres tooling can be used to do it and we will be going over how in this guide.

Reply:
👍🏼 thanks for clarifying!